File: CS通信/HTTP/Client/JavaClient/javaClient/src/UserServiceInterface/javaUserInfo.java
package UserServiceInterface;

import java.io.Serializable;

public class javaUserInfo implements Serializable {

	private String _id;
	private String _code;

	/**
	 * @return the _id
	 */
	public String get_id() {
		return _id;
	}

	/**
	 * @param _id
	 *            the _id to set
	 */
	public void set_id(String _id) {
		this._id = _id;
	}

	/**
	 * @return the _code
	 */
	public String get_code() {
		return _code;
	}

	/**
	 * @param _code
	 *            the _code to set
	 */
	public void set_code(String _code) {
		this._code = _code;
	}

}

File: CS通信/HTTP/Service/HLXMLHelper/javaUserInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace UserServiceInterface
{
    [Serializable]
    public class javaUserInfo
    {
        private string _id;

        public string Id
        {
            get { return _id; }
            set { _id = value; }
        }
        private string _code;

        public string Code
        {
            get { return _code; }
            set { _code = value; }
        }
        private string _name;

        public string Name
        {
            get { return _name; }
            set { _name = value; }
        }
        private int _age;

        public int Age
        {
            get { return _age; }
            set { _age = value; }
        }
        private int _sex;

        public int Sex
        {
            get { return _sex; }
            set { _sex = value; }
        }
    }
}
